### ISSUE TITLE
screenshot failed with extension, fallback to DevTools

### ISSUE THREAD

**Issue by lavaldi** [galenframework/galen#315]
Hi, Ivan

I have run my test with Chrome and I get this error:
[6252] [WARNING]: failed with extension screenshot, fallback to DevTools
![Image of Error](http://i.imgur.com/rKgj4A4.png)

and sometimes simply do not run the test by javascript injection:
![Image of Error](http://i.imgur.com/gpRsEql.png)

I'm using Galen version 2.0.9

Thanks for all

**Comment by ishubin** [comment]
Hi, this is weird. Why do you have such a small window? I don't think it is possible to create a screenshot with such window. Because chrome normally gives you only the viewport area in the screenshot and if it is like that, then I would expect it to fail

**Comment by lavaldi** [comment]
Sure, but every time I run the test I get that small window, what can be caused this?

**Comment by ishubin** [comment]
Well, I don't know. Never faced such issue. Are you resizing the browser window somewhere in your tests?

**Comment by lavaldi** [comment]
No, I don't resized the window.

My test: 
```
@@ parameterized
    | deviceName | tags      | size     |
    | Mobile     | mobile    | 320x568  |
    | Tablet     | tablet    | 1024x768 |
    | Desktop    | desktop   | 1280x720 |

Home page on ${deviceName} device
    selenium chrome http://dev.4a.aptitus.com/empresa/ ${size}
        inject tests/home.js
        check specs/empresa/home/home.gspec --include "${tags}"
```

I use elementary OS 0.3 Freya

**Comment by ishubin** [comment]
It happens in all tests or only in mobile tests?

**Comment by ishubin** [comment]
Oh, wait. I have just read the exception and it seems like it complains about the javascript that you are trying to inject.  Can you share the contents of tests/home.js ?

**Comment by lavaldi** [comment]
Yes, the error (small window) appears on the mobile testing, when the test runs.

The javascript error (is other error) appears when the test is not executed.

This is the home.js:
```
$('.our_clients_slider').slick('slickPause');
```

**Comment by ishubin** [comment]
I tried to use the same resolution (320x568) and ran the galen test in Chrome. It worked as expected, the screenshot is valid and galen does resize the window properly. I can't reproduce this weird behavior that you get.

**Comment by lavaldi** [comment]
:disappointed_relieved:

**Comment by hypery2k** [comment]
the above error also happens for me at customer network, but just for the first run of chrome. If I have 5 devices defined the other 4 are fine
